src/gui/addnewtorrentdialog.h

```diff
@@ -87,7 +89,7 @@ private slots:
     void reject() override;
 
 private:
-    explicit AddNewTorrentDialog(QWidget *parent = 0);
+    explicit AddNewTorrentDialog(const BitTorrent::AddTorrentParams &inParams, QWidget *parent = 0);
```

Comment:
Default value for parent does not make sense when the corresponding default gets removed from `show()`.